MD/DerivativesTemporal/ETOptionsTemporal.rdf

```diff
@@ -121,42 +123,37 @@
 	<owl:Class rdf:about="&fibo-md-derx-eto;OptionTheoreticalValue">
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-md-derx-eto;definedBy"/>
-				<owl:someValuesFrom rdf:resource="&fibo-md-temx-tem;PriceVolatilitySpecification"/>
+				<owl:onProperty rdf:resource="&fibo-fbc-dae-dbt;isBasedOn"/>
+				<owl:someValuesFrom rdf:resource="&fibo-md-derx-eto;OptionVolatility"/>
```

Comment:
That's not clear - the class itself is an orphan (was always) - we'll need options experts to help us with this one down the road, so that if we retain it, it is placed properly in the hierarchy and the relevant restrictions are applied.  I agree that it is more complicated than what's currently in the model.

Reply:
We should probably open a new issue for this change